src/main/java/randomizers/FigureSupplier.java

```diff
@@ -0,0 +1,62 @@
+package randomizers;
+
+import core.figures.Circle;
+import core.figures.IsoscelesTrapezoid;
+import core.figures.Rectangle;
+import core.figures.RightTriangle;
+import core.figures.Square;
+import java.util.Random;
+import randomizers.sourcerand.Colors;
+
+public class FigureSupplier {
+    private static final Random RANDOM = new Random();
+    private final ColorSupplier colorSupplier = new ColorSupplier();
+
+    public String getRandomFigure(int i) {
```

Comment:
let's make this method without parameters and create a random number for the switch case block here in the method